crates/sui-json-rpc/src/api/governance.rs

```diff
@@ -32,10 +33,15 @@ pub trait GovernanceReadApi {
         epoch: Option<EpochId>,
     ) -> RpcResult<SuiCommittee>;
 
-    /// Return latest SUI system state object on-chain.
+    /// (Deprecated) Return latest SUI system state object on-chain.
+    /// This is now deprecated in favor of get_latest_sui_system_state.
     #[method(name = "getSuiSystemState")]
     async fn get_sui_system_state(&self) -> RpcResult<SuiSystemStateRpc>;
 
```

Comment:
```suggestion
    /// (Deprecated) Return latest SUI system state object on-chain.
    /// This is now deprecated in favor of get_latest_sui_system_state.
    #[method(name = "getSuiSystemState", deprecated)]
    async fn get_sui_system_state(&self) -> RpcResult<SuiSystemStateRpc>;

```

nit: you can use the deprecated attribute now